sbin/patchman

```diff
@@ -520,6 +532,9 @@ def collect_args():
     parser.add_argument(
         '-e', '--errata', action='store_true',
         help='Download CentOS errata from https://cefs.steve-meier.de/')
+    parser.add_argument(
+        '-k', '--killhost',
```

Comment:
Can the long form be `--delete-hosts` to match the function it's calling?

Reply:
Rewritten per request